poc_iot_verifier/src/poc.rs

```diff
@@ -344,7 +360,12 @@ impl Poc {
         let gain = beaconer_info.gain;
         let min_rcv_signal =
             calc_fspl(tx_power, witness.frequency, witness_distance, gain).unwrap();
-        if witness.signal as f64 > min_rcv_signal {
+        tracing::debug!(
+            "signal: {:?}, min_rcv_signal: {:?}",
+            witness.signal,
+            min_rcv_signal
+        );
+        if witness.signal < min_rcv_signal as i32 {
             tracing::debug!(
```

Comment:
I guess these will all turn into actual Error returns for some new WitnessValidationError or something?

Reply:
yeah atm i have disabled the Error for RSSI check, until I get the rssi calculation updated to account for the signal being in ddbm